packages/palette/src/elements/ColumnGrid/ColumnGrid.story.tsx

```diff
@@ -0,0 +1,143 @@
+import { storiesOf } from "@storybook/react"
+import React from "react"
+import { Box } from "../Box"
+import { Text } from "../Text"
+import { Column, ColumnBreak, ColumnGrid } from "./ColumnGrid"
+
+const IPSUM =
+  "Lorem ipsum dolor sit amet consectetur adipisicing elit. Architecto deleniti recusandae ullam laudantium ut reiciendis, doloribus qui sequi id ea, ad suscipit eos placeat magnam consequatur sunt, quaerat eius saepe."
+
+const ColumnGridDebug = () => (
+  <ColumnGrid
+    position="absolute"
+    top={0}
+    left={0}
+    width="100%"
+    height="100%"
+    style={{ pointerEvents: "none" }}
+  >
+    {[...new Array(12)].map((_, i) => (
+      <Column key={i} span={1} bg="rgba(255, 0, 0, 0.05)" height="100%" />
+    ))}
+  </ColumnGrid>
+)
+
+storiesOf("Components/ColumnGrid", module)
+  .add("Real-world Example", () => {
+    return (
+      <ColumnGrid position="relative" m={2}>
+        <ColumnGridDebug />
+
+        <Column span={6} breakAfter>
+          <Text as="h1" variant="largeTitle">
+            Page Title Long Enough So As To Line Break
+          </Text>
+        </Column>
+
+        <Column span={6}>
+          <Text as="h2" variant="subtitle" mb={1}>
+            Page subtitle
+          </Text>
+
+          <Text>{IPSUM}</Text>
+        </Column>
+
+        <Column span={5} start={8}>
```

Comment:
For what it's worth I like the names `span` and `start`, this example reads very clearly to me about what it's doing.